drivers/platform/maxim/max32650/maxim_spi.c

```diff
@@ -142,6 +151,8 @@ static int32_t _max_spi_enable_ss(uint32_t id, uint32_t chip_select)
 		return -EINVAL;
 	}
 
+	spi_pins.vssel = st->init_param->vssel;
```

Comment:
i think we should do also `cs.vssel = st->init_param->vssel`. Or is the vssel for the chip select treated somewhere else?